### PR TITLE
Task06, 07 and 08 are done

### PR DESCRIPTION
**Pull request by miguex255**

### CODE REVIEW

File: Assignment4/miguex255-190322/Task06.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  print(s,p,o)
+
+
+# **TASK 6.5: Add UPM as the university where John Smith works**
+
+# In[11]:
+
+
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, VCARD.Work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/miguex255-190322/Task06.py]: vcard work does not exist in vcard. Create your own in your namespace

File: Assignment4/miguex255-190322/Task07.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  print(s)
+  for s1,p1,o1 in g.triples((None,RDFS.subClassOf,s)):
+      print(s1)
+print('\n')
+
+q1 = prepareQuery('''
+  SELECT 
+    ?s
+  WHERE { 
+    ?s (rdfs:subClassOf/rdfs:subClassOf*) ns:Person.
```
Comment (dgarijo) [inline Assignment4/miguex255-190322/Task07.py]: here 1 subclass of is enough

File: Assignment4/miguex255-190322/Task08.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+from rdflib.namespace import RDF
+from rdflib.plugins.sparql import prepareQuery
+ns = Namespace("http://data.org#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+
+# In[4]:
+
+
+#Extraemos name, family name y email del segundo grafo y lo añadimos al primero
```
Comment (dgarijo) [inline Assignment4/miguex255-190322/Task08.py]: They have to be from the class Person.

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by miguex255** [comment]
Good evening teacher, sorry about the commit including the notebooks checkpoint folder, I forgot to delete it. It won't happen again.

**Comment by dgarijo** [comment]
@miguex255 you still have the checkpoints. And you have modified the class material notebooks. I cannot accept it as it is, sorry

**Comment by miguex255** [comment]
Teacher, to try to fix it, do I have to create another pull-request and close this one.?

**Comment by dgarijo** [comment]
no, just commit new changes as you have done.

**Comment by dgarijo** [comment]
You are still submitting changes to the class notebooks. I cannot accept the PR :(

**Comment by dgarijo** [comment]
You are still adding notebook checkpoints and changes to notebooks. This PR is not acceptable

**Pull request closed without merging**